ui/app/components/VerticalBarChart.tsx

```
@@ -28,14 +28,14 @@ interface VerticalBarChartProps {
   xAxisTitle: string;
 }
 
-const getBarColor = (vendor: string, defaultColor: string) => {
+const getBarColor = (vendor: string) => {
   switch (vendor.toLowerCase()) {
     case "falkordb":
-      return "#FF66B3";
+      return getComputedStyle(document.documentElement).getPropertyValue("--FalkorDB-color").trim();
     case "neo4j":
-      return "#0B6190";
+      return getComputedStyle(document.documentElement).getPropertyValue("--Neo4j-color").trim();;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)